content/applications/sales/sales/products_prices/prices/currencies.rst

```diff
@@ -1,49 +1,237 @@
-=================================
-How to sell in foreign currencies
-=================================
+==================
+Foreign currencies
+==================
+
+With Odoo, pricelists can be used to manage prices in a number of foreign currencies. Specifically,
+Odoo has the ability to work with 167 total currencies. In order to use multiple currencies in Odoo
+*Sales*, the *Accounting* application **must** be installed.
+
+Settings
+========
 
-Pricelists can also be used to manage prices in foreign currencies.
+Once the *Accounting* app is installed, foreign currencies can be viewed, edited, and added to the
```

Comment:
- tense mismatch
- would condense here, the other two actions are implied (and seem out of order anw?)
```suggestion
Once the *Accounting* app has been installed, foreign currencies can be added to the
```